interview-questions.md

```diff
@@ -7,53 +7,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a hash in Ruby?
 
-  Your answer:
+  Your answer: A hash in Ruby is a container of data set in a key:value pair. 
 
-  Researched answer:
+  Researched answer: In Ruby a hash is a container/collection of key: value pairs, with the value able to be referenced by its unique key. 
```

Comment:
While this is correct, I would really like to see you elaborate or expand further on this in the future.  One sentence answers will not be a great representation of your knowledge in an interview.